src/mono/mono/metadata/assembly.c

```diff
@@ -1138,6 +1138,15 @@ netcore_load_reference (MonoAssemblyName *aname, MonoAssemblyLoadContext *alc, M
 		}
 	}
 
+	// For compatibility with CoreCLR, invoke the Resolving event in the default ALC first whenever loading
```

Comment:
@uweigand Could you also update step 8 in the big comment that summarizes the algorithm.

Reply:
Done.